src/components/Icon/iconMapping.ts

```diff
@@ -25,6 +26,7 @@ import IconActionSearch from './components/IconActionSearch';
 import IconActionSettings from './components/IconActionSettings';
 import IconActionSubmit from './components/IconActionSubmit';
 import IconActionTickCircle from './components/IconActionTickCircle';
+import IconActionTickCircleFilled from './components/IconActionTickCircleFilled';
```

Comment:
Ignore [this comment](https://github.com/wellcometrust/design-system/pull/25/files#r789746827) I can see _TickCircle_ is the type and _Filled_ is the variant/state